### PR TITLE
Adds sports

### PR DESCRIPTION
**Pull request by tortellinitony**
:cl: Tortellini Tony
add: Bowling and wrestling kits are now available.
tweak: The hockey, bowling, and wrestling kits can be purchased under the "sports kit" item in the badassery section of an uplink.
/:cl:

### CODE REVIEW

**Comment by tortellinitony** [comment]
Wait for the hockey PR to merge first.

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -5,6 +5,7 @@
 	var/datum/thrownthing/throwing = null
 	var/throw_speed = 2 //How many tiles to move per ds when being thrown. Float values are fully supported
 	var/throw_range = 7
+	var/unlimitedthrow = 0 //HOLY SHIT
```
Comment (ghost) [inline code/game/atoms_movable.dm]: Please use `TRUE` or `FALSE`

File: code/game/objects/objs.dm [hunk 2]
```diff
@@ -4,6 +4,7 @@
 	var/crit_fail = 0
 	animate_movement = 2
 	var/throwforce = 0
+
```
Comment (ghost) [inline code/game/objects/objs.dm]: This file contains no changes?

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+/obj/item/clothing/under/bowling
+	name = "bowling jersey"
+	desc = "The latest in kingpin fashion."
+	icon = 'icons/hippie/obj/sports.dmi'
+	icon_state = "bowlinguniform"
+	item_color = "bowling"
+	resistance_flags = FIRE_PROOF |  ACID_PROOF
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 100, acid = 100)
+	can_adjust = 0
+	var/canbowl = 1
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: Please use `TRUE` or `FALSE`

I would also suggest changing this to `can_bowl`

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+/obj/item/clothing/under/bowling/proc/canbowl()
+	canbowl = 1
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: Please use `TRUE` or `FALSE`

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+	desc = "The latest in kingpin fashion."
+	icon = 'icons/hippie/obj/sports.dmi'
+	icon_state = "bowlinguniform"
+	item_color = "bowling"
+	resistance_flags = FIRE_PROOF |  ACID_PROOF
+	armor = list(melee = 0, bullet = 0, laser = 0, energy = 0, bomb = 0, bio = 0, rad = 0, fire = 100, acid = 100)
+	can_adjust = 0
+	var/canbowl = 1
+
+/obj/item/clothing/under/bowling/proc/canbowl()
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: I would suggest changing this to `can_bowl` but I'm not sure if we really need a proc for this, you can just check the vars on clothing no?

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+		playsound(src,'sound/misc/bowlhit.ogg',60,0)
+		unspin()
+		return
+	else //Caught and not spinning or something else weird.
+		unspin()
+		return ..()
+
+/obj/item/weapon/twohanded/bowling/proc/unspin()
+	icon_state = "bowling_ball"
+	unlimitedthrow = 0
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: Please use `TRUE` or `FALSE`

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+	prowielded = 0
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: Please use `TRUE` or `FALSE`

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+		if(user.w_uniform && istype(user.w_uniform, /obj/item/clothing/under/bowling))
+			var/obj/item/clothing/under/bowling/bowling = user.w_uniform
+			if(!bowling.canbowl)
+				user << "<span class='warning'>Your coach always said that you need to wait at least a second between bowls!</span>"
+				unspin()
+			else
+				wield(user)
+				if(!wielded)
+					return
+				bowling.canbowl = 0
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: Please use `TRUE` or `FALSE`

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+				addtimer(CALLBACK(bowling, /obj/item/clothing/under/bowling.proc/canbowl), 10) //1 second to avoid spam
+				unlimitedthrow = 1
+				prowielded = 1
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: Please use `TRUE` or `FALSE`

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+			if(!bowling.canbowl)
+				user << "<span class='warning'>Your coach always said that you need to wait at least a second between bowls!</span>"
+				unspin()
+			else
+				wield(user)
+				if(!wielded)
+					return
+				bowling.canbowl = 0
+				addtimer(CALLBACK(bowling, /obj/item/clothing/under/bowling.proc/canbowl), 10) //1 second to avoid spam
+				unlimitedthrow = 1
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: Please use `TRUE` or `FALSE`

File: code/modules/hippie/code/game/objects/bowling.dm [hunk 11]
```diff
@@ -0,0 +1,105 @@
...
+	icon = 'icons/hippie/obj/sports.dmi'
+	icon_state = "bowling_ball"
+	desc = "A heavy, round device used to knock pins (or people) down."
+	force_unwielded = 3
+	force_wielded = 6
+	w_class = 3.0
+	throwforce = 0
+	throw_range = 1
+	throw_speed = 1
+	var/prowielded = 0
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/bowling.dm]: Please use `TRUE` or `FALSE`

I would also suggest changing this to `pro_wielded`

File: code/modules/hippie/code/game/objects/wrestling.dm [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+	flags_cover = GLASSESCOVERSEYES
+	visor_flags_inv = HIDEEYES
+	glass_colour_type = /datum/client_colour/glass_colour/gray
+
+/obj/item/clothing/under/wrestling
+	name = "wrestling unitard"
+	icon = 'icons/hippie/obj/sports.dmi'
+	desc = "Doesn't do much, but looks stylish as Hell."
+	icon_state = "unitard"
+	can_adjust = 0
```
Comment (ghost) [inline code/modules/hippie/code/game/objects/wrestling.dm]: Please use `TRUE` or `FALSE`

**Review by ghost — CHANGES_REQUESTED** [review]
Fuck Yes.
Just needs some small changes.

**Review by ghost — APPROVED** [review]

**Pull request merged**